docs/getting-started/settings.en.md

```diff
@@ -150,13 +155,23 @@ Set the time it takes for Krux to automatically shut down. This feature not only
 
 **Note**: Devices without batteries and power management will not shut down but reboot, which will also unload keys.
 
+<div style="clear: both"></div>
+
 #### TC Flash Hash at Boot
+<img src="../../img/maixpy_m5stickv/settings-tc-boot-250.png" align="right" class="m5stickv">
+<img src="../../img/maixpy_amigo/settings-tc-boot-300.png" align="right" class="amigo">
+
 Chose if you would like to run [Tamper Check Flash Hash](features/tamper-detection.md) every time the device is powered on.
 
 Activating *TC Flash Hash* at boot helps prevent unauthorized use by requiring the *TC Code*. But is important to note, unlike a PIN, the *TC Code* does not provide access control over USB. This means that the device's memory remains accessible via USB, allowing it to be flashed with firmware that does not require the *TC Code*.
 
+<div style="clear: both"></div>
+
 #### Hide Mnemonics
-When `True`, your device will not display private key data or backup tools when a key is loaded. It will only show public key information and allow signing operations.
+<img src="../../img/maixpy_m5stickv/settings-hide-mnemonic-250.png" align="right" class="m5stickv">
+<img src="../../img/maixpy_amigo/settings-hide-mnemonic-300.png" align="right" class="amigo">
+
+When `True`, Krux will hide the [New Mnemonic](./usage/generating-a-mnemonic.md) menu and hide the words when [Loading a Mnemonic](./usage/loading-a-mnemonic.md). It will also hide the words when using [BIP85 to create a BIP39 Mnemonic](./usage/navigating-the-main-menu.md#bip85) and disable the [Backup Mnemonic](./usage/navigating-the-main-menu.md#backup-mnemonic) menu.
```

Comment:
Krux will disable the [New Mnemonic] ...

Reply:
Done, thx for the suggestion :+1: